code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -365,6 +365,7 @@
 	if(can_be_disabled)
 		update_disabled()
 
+	new_limb_owner.bodyparts = sort_list(new_limb_owner.bodyparts, /proc/cmp_bodypart_by_body_part_asc)
```

Comment:
Leave a comment here as to why this exists so someone doesnt fuck it up later

Reply:
True.